Bringing the i18n page up to date (#1289)
mattermost/mattermost-handbook: contributors/join-us/localization.md

### Confirm whether the language you want to translate is one we officially support or is a work-in-progress (WIP) language. To contribute to the supported languages you need extra permissions, this is a safety measure. 
Requesting these permissions can be done in the [Mattermost localization channel](https://community.mattermost.com/core/channels/localization) or with a direct message to Carrie Warner or Tom De Moor.

Only supported languages are available in-product by selecting **Settings > Display > Language**. Any language that isn't officially supported is considered a work-in-progress language. Work-in-progress languages become officially supported languages when the following two conditions are met:
* The (WIP) language reaches or exceeds the Beta translation quality threshold across three consecutive releases; and,
* A language expert is available for at least six months of maintenance updates.

See the section on [translation quality](https://handbook.mattermost.com/contributors/join-us/localization#translation-quality) for further details.

> **Tip:** We encourage translators to add additional languages. We warm welcome new languages or regional variants.

Reviewer: ```suggestion
**Tip:** We encourage translators to help us support additional languages and regional variants.
```

## Administrative tasks

> Translation server admins must grant all new translators with specific permissions to enable translators to start working on Mattermost strings for the official supported languages. Translation server admins can reach the server's administration tools by going to https://translate.mattermost.com/admin/.

Reviewer: ```suggestion
Translation server admins must grant all new translators specific Weblate permissions before translators can start contributing to official supported languages. Translation server admins can reach the server's administration tools by going to https://translate.mattermost.com/admin/.
```
